contracts/protocol/tokenization/NTokenApeStaking.sol

```diff
@@ -62,12 +76,13 @@ abstract contract NTokenApeStaking is NToken {
     ) external override onlyPool nonReentrant {
         uint256[] memory tokenIds = new uint256[](1);
         tokenIds[0] = tokenId;
-        ApeStakingLogic.executeAutoWithdraw(
-            _ERC721Data.owners,
-            _apeCoinStaking,
-            POOL_ID(),
-            tokenIds,
-            from
+        require(
```

Comment:
we shouldn't prevent trasnfer on liquidation. instead we should unstake and repay then trasnfer.

Reply:
Current design is you must `unstake_and_repay` before actual liquidation.  I think this way have a better protection for user's NFT, because `unstake_and_repay` may be can trigger HF up to 1 then need to liquidate NFT.